tools/cli/go-whisk/whisk/activation.go

```diff
@@ -41,6 +42,7 @@ type Activation struct {
     End          int64  `json:"end"`                    // Since a 0 is a valid value from server, don't omit
     Response     `json:"response,omitempty"`
     Logs         []string `json:"logs,omitempty"`
+    Annotations *json.RawMessage    `json:"annotations,omitempty"`
```

Comment:
nit: fix the whitespacing? just remove it down to 1 here or adjust every line to match.